debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlDefaultValueTest.java

```diff
@@ -24,7 +24,7 @@
 public class MySqlDefaultValueTest extends BinlogDefaultValueTest<MySqlValueConverters, MySqlAntlrDdlParser> {
     @Override
     protected MySqlAntlrDdlParser getDdlParser(MySqlValueConverters valueConverter) {
```

Comment:
```suggestion
    protected MySqlAntlrDdlParser getDdlParser() {
```
Same as my former comment.